code/modules/clothing/glasses/glasses.dm

```diff
@@ -279,6 +279,16 @@
 	icon_state = "redglasses"
 	item_state = "redglasses"
 
+/obj/item/clothing/glasses/godeye
+	name = "eye of god"
+	desc = "A strange eye, said to have been torn from an omniscient creature that used to roam the wastes."
+	icon_state = "godeye"
+	item_state = "godeye"
+	vision_flags = SEE_TURFS|SEE_MOBS|SEE_OBJS
+	darkness_view = 8
+	scan_reagents = 1
+	flags = 2

```

Comment:
`flags = NODROP` since 2 is NODROP.

ALthough are you sure you want the item to be not be unequippable?